src/components/molecules/SearchBar.module.css

```diff
@@ -1,17 +1,70 @@
-.form {
-  margin-bottom: var(--spacer);
-  width: 100%;
-  max-width: 30rem;
+.search {
+  display: flex;
+  flex: 1 0 auto;
+  align-self: stretch;
+}
+.button {
+  padding: calc(var(--spacer) / 6) calc(var(--spacer) / 3);
+  cursor: pointer;
+  border: 1px solid var(--border-color);
+  border-radius: var(--border-radius);
+  background-color: var(--background-content);
+  border-left: none;
+  white-space: nowrap;
+  min-width: 4rem;
+}
+
+.button:hover,
+.button:focus {
+  color: var(--brand-white);
+  text-decoration: none;
+  transform: translate3d(0, -0.05rem, 0);
+  box-shadow: 0 12px 30px 0 rgba(0, 0, 0, 0.1);
+}
+
+.input {
+  height: 36px !important;
+  border-top-right-radius: 0px;
+  border-bottom-right-radius: 0px;
+}
+.searchInput {
+  flex-grow: 2;
+  margin-bottom: 0px;
+}
+/* @media screen and (min-width: 600px) {
```

Comment:
Not sure why styling has been added an commented out in this file? Might be better just to remove it